### ISSUE TITLE
ReferenceError: React is not defined

### ISSUE THREAD

**Issue by GrzegorzKastelik** [dev-javascript/string-to-react-component#12]
When you try to use this in some scenarion with Next.js and client components you will get error 'ReferenceError: React is not defined'

It's can be fix in **ctx.tsx** adding global.React = React; when window not type 'object'

```
if (typeof window === 'object') {
    window.React = window.React || React;
  } else {
  global.React = React;
}
```

Can be added upgrade like this?

**Comment by dev-javascript** [comment]
Thanks, Yes
A new version will be published in the next few days and React will not be global anymore.

**Comment by dev-javascript** [comment]
Has been fixed since version `4.0.0`
